packages/serverless/src/domain/use-case/schema.ts

```diff
@@ -1,126 +1,126 @@
 import {
   PROVABLE_TYPE_MAP,
-  Schema,
+  Schema as O1Schema,
```

Comment:
Not prefer alias

Reply:
it conflicts with the class Schema so I had to alias it
https://github.com/orochi-network/zkDatabase/blob/e6c569fc5454d6092d0b2106e0927c4fdcc2d647/packages/serverless/src/domain/use-case/schema.ts#L58-L60

Reply:
We don't have schema in serverless, we better rename schema in serverless.

Reply:
fixed